datacenter/ucp/2.1/guides/admin/configure/external-auth/index.md

```diff
@@ -1,63 +1,77 @@
 ---
-description: Learn how to integrate UCP with an LDAP service, so that you can manage
-  users from a single place.
-keywords: LDAP, authentication, user management
-title: Integrate with LDAP
+description: Learn how to integrate UCP with an LDAP service, so that you can
+  manage users from a single place.
+keywords: LDAP, directory, authentication, user management
+title: Integrate with an LDAP Directory
 ---
 
 Docker UCP integrates with LDAP services, so that you can manage users from a
-single place.
+single place. Many enterprise organizations have an existing directory service
```

Comment:
Sentences that start with "Many" feel a bit boring, since you would not speak like that in a face to face conversation. Maybe
"...you can manage from a single place. This way you can provision users and team membership in a centralized way and it will automatically propagate that information to UCP and DTR."